src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -23,7 +23,7 @@ internal enum Stamp
             SetToMaxUsed = 3,
         }
 
-        private readonly ArrayList m_list = new ArrayList();
+        private readonly List<Cookie> m_list = new List<Cookie>();
```

Comment:
@ViktorHofer, This type is `[Serializable]`. Is it OK to change this field's type from `ArrayList` to `List<Cookie>`?

Reply:
Unfortunately not.

Reply:
I fixed it.

Reply:
@ViktorHofer, are we missing a test then that would have caught that? Or did a test fail?

Reply:
Sorry, I didn't check the CI before the change but a test must have failed. I just checked the BinaryFormatterTestData to make sure, there shouldn't be a test hole.

Reply:
Found it: https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_netcoreapp+CGroup_Debug+AGroup_x64+TestOuter_false_prtest/12974/

Reply:
Test did fail, everything's good :)

Reply:
Great, thanks.